bridge/client/app/_components/ktb-notification/ktb-notification.component.scss

```diff
@@ -0,0 +1,32 @@
+@import '../../../variables';
+@import '~@dynatrace/barista-components/core/src/style/variables';
+
+dt-alert {
+  max-width: 600px;
```

Comment:
I am just thinking, if we should give it a fixed size. It could look weird if we have 2 stacked that differ in size.